.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: ["plugin:vue/vue3-essential", "eslint:recommended", "@vue/prettier"],
+  parserOptions: {
+    parser: "babel-eslint",
+  },
+  rules: {
+    "no-console": process.env.NODE_ENV === "production" ? "warn" : "off",
+    "no-debugger": process.env.NODE_ENV === "production" ? "warn" : "off",
+  },
+};
```

Comment:
Oh no, are we going to use semicolons in our js files?

Reply:
These are all generated by Vue CLI, I haven't touched any of these files yet, but I can delete the semicolons if You want.

Reply:
I don't care, this is something that linter should do.

Reply:
You can create `Prettier` configuration file called `.prettierrc`. Now it's probably using default config from Vue.